### PR TITLE
User layout changes

### PR DESCRIPTION
**Pull request by clash99**
### Proposed changes in this pull request

- Completes #677 
- Login screen added messaging for redirected anonymous users
- Modified layout for orgs and projects for anonymous and registered users without project permissions
- Added import data to project "Add location" button
- H1 and H2 css tweaks for long titles 
- Responsive css tweaks for smallest screens

### When should this PR be merged

- Anytime, for Sprint 11

### Risks

- None

### CODE REVIEW

File: cadasta/templates/organization/project_wrapper.html [hunk 1]
```diff
@@ -64,11 +66,10 @@
           <span class="sr-only">{%trans "Toggle add" %}</span>
         </button>
         <ul class="dropdown-menu">
-          <!--<li><a href="#">{% trans "Add party" %}</a></li>
-          <li><a href="#">{% trans "Add relationship" %}</a></li>-->
+          <!--<li><a href="#">{% trans "Add party" %}</a></li>-->
```
Comment (oliverroick) [inline cadasta/templates/organization/project_wrapper.html]: Can we remove this?

File: cadasta/templates/organization/project_wrapper.html [hunk 2]
```diff
@@ -108,8 +109,8 @@
         <span class="icon"></span>
         <span class="title">{% trans "Overview" %}</span>
       </a>
-    </li>
-    <li class="map">
+    </li><!--
```
Comment (oliverroick) [inline cadasta/templates/organization/project_wrapper.html]: Can we remove the empty comment?

File: cadasta/core/static/css/main.scss [hunk 3]
```diff
@@ -470,10 +467,10 @@ header {
       white-space: nowrap;
       overflow: hidden;
       text-overflow: ellipsis;
-      &:hover{
-        overflow: visible; 
-        white-space: normal; 
-      }
+      //&:hover{
```
Comment (clash99) [inline cadasta/core/static/css/main.scss]: Seems like the css isn't being updated here.  Gray bar should not appear and login page layout should be wider as in screenshots (https://github.com/Cadasta/cadasta-platform/issues/677).  Removed unused hover comments.

File: cadasta/templates/organization/project_wrapper.html (hunk 1 above)
Comment (clash99) [inline cadasta/templates/organization/project_wrapper.html]: Yep

File: cadasta/templates/organization/project_wrapper.html (hunk 2 above)
Comment (clash99) [inline cadasta/templates/organization/project_wrapper.html]: Yep

**Comment by clash99** [comment]
Updated branch with minimal changes but still not sure why main.css doesn't seem to be updated with most recent main.scss.  Let me know if you are still having the issue.

File: cadasta/core/static/css/main.scss (hunk 3 above)
Comment (oliverroick) [inline cadasta/core/static/css/main.scss]: You are right, maybe the CSS was still cached in my browser. It looks as expected now.

**Comment by oliverroick** [comment]
I added `is_project_member` to the template context, which is set to `True` is the user is a superuser, organization admin, project manager, data collector or project user. I also changed `project_wrapper.html` accordingly. @clash99 can you check if this is all ok, this should be ready for approval then.

**Comment by clash99** [comment]
Looks good Oliver - thank you for your help.  

There seems to be some unrelated permissions issues that make testing this a bit tricky but I will file on QA day if they still exist.

**Pull request merged**